tests/spread/plugins/v2/snaps/cmake-hello-ninja/snap/snapcraft.yaml

```diff
@@ -0,0 +1,23 @@
+name: cmake-hello
```

Comment:
this test is not added to the v2 task.yaml and will likely fail as the name does not match the directory name holding this project